core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -509,11 +508,10 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
     val bmId1 = BlockManagerId("id1", localHost, 1)
     val bmId2 = BlockManagerId("id2", localHost, 2)
     val bmId3 = BlockManagerId("id3", otherHost, 3)
-    when(bmMaster.getLocations(mc.any[BlockId])).thenReturn(Seq(bmId1, bmId2, bmId3))
```

Comment:
why remove it? the `BlockManager#getLocations` still exist

Reply:
Agreed, will revert it back.